sql/core/src/test/resources/sql-tests/inputs/explain.sql

```diff
@@ -4,7 +4,7 @@ CREATE table  explain_temp2 (key int, val int) USING PARQUET;
 CREATE table  explain_temp3 (key int, val int) USING PARQUET;
 
 SET spark.sql.codegen.wholeStage = true;
-
+SET spark.sql.adaptive.enabled = false;
```

Comment:
since we are touchinng it, let's use the right approach. We should put this line at the beginning of this file
```
--SET spark.sql.codegen.wholeStage = true
--SET spark.sql.adaptive.enabled = false
```